preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -559,31 +557,29 @@ def format_frameshift(result):
     * Makes the range [] have an inclusive start and inclusive end
     (the default in nextclade is exclusive end)
     """
-    if result == "[]":
+    if input == "[]":
         return ""
-    result = result.replace("'", '"')
-    frame_shifts = json.loads(result)
+
+    def range_string(_start: str | int, _end: str | int) -> str:
+        """Converts 0-indexed exclusive range to 1-indexed inclusive range string"""
+        start = int(_start) + 1
+        end = int(_end)
+        if end > start:
+            return f"{start}-{end}"
+        return str(start)
+
+    frame_shifts = json.loads(input.replace("'", '"'))  # Why is replace needed?
```

Comment:
This is required by json.loads to recognize input as a json object and convert to a dictionary